src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder stringBuilder = new StringBuilder("Report for period ");
+        stringBuilder
+                .append(dateFrom)
+                .append(" - ")
+                .append(dateTo);
+        LocalDate localDateFrom = LocalDate.parse(dateFrom, FORMATTER);
+        LocalDate localDateTo = LocalDate.parse(dateTo, FORMATTER);
+        for (String name : names) {
+            int salary = 0;
+            for (String record : data) {
+                String[] recordData = record.split(" ");
+                LocalDate recordDate = LocalDate.parse(recordData[0], FORMATTER);
```

Comment:
Any magic numbers should be constants

Indexes should be constant with proper names.